src/Compilers/CSharp/Portable/Symbols/SymbolExtensions.cs

```diff
@@ -108,7 +108,7 @@ public static LocalizableErrorArgument GetKindText(this Symbol symbol)
         internal static NamespaceOrTypeSymbol? ContainingNamespaceOrType(this Symbol symbol)
         {
             var containingSymbol = symbol.ContainingSymbol;
-            if ((object)containingSymbol != null)
+            if ((object?)containingSymbol != null)
```

Comment:
Consider annotating line 63 as well: `for (NamedTypeSymbol? current = subType;`

Reply:
added